lib/task_helpers.rb

```diff
@@ -1,213 +1,108 @@
     end
 
-    def self.move_alias(alias_name, index_name, force_delete=false)
-      unless index_name != alias_name
-        raise ArgumentError, "Can't point alias [#{alias_name}] to an index of the same name."
-      end
-      unless exists_index(index_name)
-        raise ArgumentError, "Can't point alias to non-existent index [#{index_name}]."
-      end
-
-      # You cannot use an alias name if an index of the same name (that is not an alias) already exists.
-      # This could happen if the index was auto-created before the alias was properly set up.  In this
-      # case, we either warn the user or delete the already existing index.
-      if exists_index(alias_name) and not exists_alias(alias_name)
-        if force_delete
-          self.delete_index(alias_name)
-        else
-          raise ArgumentError, "Can't create alias [#{alias_name}] because there is already an " +
-              "auto-generated index of the same name. Try again with force_delete=true to first " +
-              "delete this pre-existing index."
-        end
-      end
-
-      actions = [
-          {add: {index: index_name, alias: alias_name}}
-      ]
-
-      begin
-        response = Elasticsearch::Model.client.indices.get_alias(name: alias_name)
-        if response.length
-          actions.unshift({remove: {index: response.keys.join(','), alias: alias_name}})
-        end
-      rescue Elasticsearch::Transport::Transport::Errors::NotFound
-        # NOTE (CCB): Future versions of the Elasticsearch client support the ignore parameter,
-        # that can be used to ignore 404 errors.
-      end
-
-      body = {actions: actions}
-      Elasticsearch::Model.client.indices.update_aliases(body: body)
-      LOG.info "Alias [#{alias_name}] now points to index [#{index_name}]."
-    end
-
-    def self.refresh_index(name)
-      Elasticsearch::Model.client.indices.refresh(index: name)
-    end
-
-    def self.initialize_index(alias_name, force_new_index)
-      # When force_new_index is true, a fresh index will be created for the alias,
-      # even if it already exists.
-      if force_new_index or not exists_alias(alias_name)
-        index_name = create_index()
-        # WARNING: if an index exists with the same name as the intended alias, it
-        #   will be deleted.
-        move_alias(alias_name, index_name, force_delete: true)
+    def self.initialize_indices(force_new_index)
+      # When force_new_index is true, fresh indices will be created even if it already exists.
+      if force_new_index or not exists_indices
+        create_indices
       else
-        LOG.info "Skipping initialization. The 'content' alias already exists. If 'rake search:validate_index' indicates "\
-          "a problem with the mappings, you could either use 'rake search:rebuild_index' to reload from the db or 'rake "\
+        LOG.info "Skipping initialization. Indices already exists. If 'rake search:validate_indices' indicates "\
```

Comment:
`Indices already exist`? Got extra S.

Reply:
What is `rake`?

Reply:
indicates. extra s too?

Reply:
Thanks) [Rake](https://www.rubyguides.com/2019/02/ruby-rake/). `rake search:validate_index` is a command

Reply:
> is a command

cool